### PR TITLE
src/sage/schemes: Doctest cosmetics

### PR DESCRIPTION
**Pull request by mkoeppe**
Standard reformatting of doctests and their outputs

Split out from https://github.com/sagemath/sage/pull/35095

### :memo: Checklist

- [ ] The title is concise and informative.
- [ ] The description explains in detail what this PR is about.
- [ ] I have linked a relevant issue or discussion.
- [ ] I have created tests covering the changes.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/sage/schemes/elliptic_curves/kraus.py [hunk 1]
```diff
@@ -921,14 +931,14 @@ def semi_global_minimal_model(E, debug=False):
...
     OUTPUT:
 
-    A tuple (Emin,I) where Emin is an elliptic curve which is either a
+    A tuple ``(Emin, I)`` where Emin is an elliptic curve which is either a
     global minimal model of E if one exists (i.e., an integral model
-    which is minimal at every prime), or a semin-global minimal model
+    which is minimal at every prime), or a semi-global minimal model
     (i.e., an integral model which is minimal at every prime except
     one).  I is the unit ideal of Emin is a global minimal model, else
     is the unique prime at which Emin is not minimal.  Thus in all
```
Comment (GiacomoPope) [inline src/sage/schemes/elliptic_curves/kraus.py]:
```suggestion
    A tuple ``(Emin, I)`` where ``Emin`` is an elliptic curve which is either a
    global minimal model of ``E`` if one exists (i.e., an integral model
    which is minimal at every prime), or a semi-global minimal model
    (i.e., an integral model which is minimal at every prime except
    one).  I is the unit ideal of ``Emin`` is a global minimal model, else
    is the unique prime at which ``Emin`` is not minimal.  Thus in all
```

Comment (GiacomoPope) [reply src/sage/schemes/elliptic_curves/kraus.py]: Add backticks around ``E`` and ``Emin`` to match the other parts of this change

Comment (GiacomoPope) [reply src/sage/schemes/elliptic_curves/kraus.py]: Of course we could also defer to Latex here, this is a style choice I suppose.

File: src/sage/schemes/elliptic_curves/period_lattice.py [hunk 2]
```diff
@@ -713,7 +713,7 @@ def _compute_periods_complex(self, prec=None, normalise=True):
         - `prec` (int or ``None`` (default)) -- floating point precision (in bits); if None,
```
Comment (GiacomoPope) [inline src/sage/schemes/elliptic_curves/period_lattice.py]:
```suggestion
        - `prec` (int or ``None`` (default)) -- floating point precision (in bits); if ``None``,
```

File: src/sage/schemes/elliptic_curves/ell_point.py [hunk 3]
```diff
@@ -3277,10 +3277,10 @@ def non_archimedean_local_height(self, v=None, prec=None,
           of self at `v`. If `v` is None, return the total
```
Comment (GiacomoPope) [inline src/sage/schemes/elliptic_curves/ell_point.py]:
```suggestion
          of self at `v`. If `v` is ``None``, return the total
```

File: src/sage/schemes/elliptic_curves/ell_point.py [hunk 4]
```diff
@@ -3277,10 +3277,10 @@ def non_archimedean_local_height(self, v=None, prec=None,
...
           non-archimedean contribution to the global height.
 
-        - ``prec`` -- integer, or None (default). The precision of the
-          computation. If None, the height is returned symbolically.
+        - ``prec`` -- integer, or ``None`` (default). The precision of the
+          computation. If ``None``, the height is returned symbolically.
 
-        - ``weighted`` -- boolean. If False (default), the height is
+        - ``weighted`` -- boolean. If ``False`` (default), the height is
           normalised to be invariant under extension of `K`. If True,
```
Comment (GiacomoPope) [inline src/sage/schemes/elliptic_curves/ell_point.py]:
```suggestion
          normalised to be invariant under extension of `K`. If ``True``,
```

File: src/sage/schemes/elliptic_curves/kraus.py [hunk 5]
```diff
@@ -108,15 +108,15 @@ def c4c6_model(c4, c6, assume_nonsingular=False):
...
     - ``c4``, ``c6`` -- elements of a number field
 
-    - ``assume_nonsingular`` (boolean, default False) -- if True,
+    - ``assume_nonsingular`` (boolean, default ``False``) -- if ``True``,
       check for integrality and nosingularity.
 
     OUTPUT:
 
     The elliptic curve with a-invariants [0,0,0,-c4/48,-c6/864], whose
     c-invariants are the given c4, c6.  If the supplied invariants are
```
Comment (GiacomoPope) [inline src/sage/schemes/elliptic_curves/kraus.py]: General comment, should we do `c_4` for latex, double backticks for code or leave c4 as c4 in plaintest? This applies to many functions in this file.

File: src/sage/schemes/elliptic_curves/kraus.py [hunk 6]
```diff
@@ -582,18 +589,18 @@ def check_Kraus_local_2(c4, c6, P, a1=None, assume_nonsingular=False):
...
+    - ``assume_nonsingular`` (boolean, default ``False``) -- if ``True``,
+      check for integrality and nonsingularity.
 
     OUTPUT:
 
-    Either (False, 0, 0) if Kraus's conditions fail, or (True, a1,
-    a3) if they pass, in which case the elliptic curve which is the
+    Either ``(False, 0, 0)`` if Kraus's conditions fail, or ``(True, a1,
+    a3)`` if they pass, in which case the elliptic curve which is the
     (a1**2/12,a1/2,a3/2)-transform of [0,0,0,-c4/48,-c6/864] is
```
Comment (GiacomoPope) [inline src/sage/schemes/elliptic_curves/kraus.py]: Same as the above, wont keep mentioning it, but these arrays of values feel as though they should be in double backticks to me.

File: src/sage/schemes/elliptic_curves/kraus.py [hunk 7]
```diff
@@ -674,13 +681,13 @@ def check_Kraus_local(c4, c6, P, assume_nonsingular=False):
...
-      check for integrality and nosingularity.
+    - ``assume_nonsingular`` (boolean, default ``False``) -- if ``True``,
+      check for integrality and nonsingularity.
 
     OUTPUT:
 
-    Tuple: either (True,E) if there is a Weierstrass model E integral
-    at P and with invariants c4, c6, or (False, None) if there is
+    Tuple: either ``(True, E)`` if there is a Weierstrass model E integral
+    at P and with invariants c4, c6, or ``(False, None)`` if there is
```
Comment (GiacomoPope) [inline src/sage/schemes/elliptic_curves/kraus.py]:
```suggestion
    Tuple: either ``(True, E)`` if there is a Weierstrass model `E` integral
    at `P` and with invariants `c_4`, `c_6`, or ``(False, None)`` if there is
```

Comment (GiacomoPope) [reply src/sage/schemes/elliptic_curves/kraus.py]: Here i move these invariants to format as Latex, but maybe this is not what we want

File: src/sage/schemes/elliptic_curves/kraus.py [hunk 8]
```diff
@@ -746,12 +753,12 @@ def check_Kraus_global(c4, c6, assume_nonsingular=False, debug=False):
...
-    - ``assume_nonsingular`` (boolean, default False) -- if True,
-      check for integrality and nosingularity.
+    - ``assume_nonsingular`` (boolean, default ``False``) -- if ``True``,
+      check for integrality and nonsingularity.
 
     OUTPUT:
 
-    Either False if Kraus's conditions fail, or, if they pass, an
+    Either ``False`` if Kraus's conditions fail, or, if they pass, an
     elliptic curve E which is integral and has c-invariants c4,c6.
```
Comment (GiacomoPope) [inline src/sage/schemes/elliptic_curves/kraus.py]:
```suggestion
    elliptic curve `E` which is integral and has c-invariants `c_4`,`c_6`
```

File: src/sage/schemes/elliptic_curves/period_lattice.py [hunk 9]
```diff
@@ -526,9 +526,9 @@ def normalised_basis(self, prec=None, algorithm='sage'):
         - ``prec`` (default: ``None``) -- precision in bits (default
           precision if ``None``).
 
-        - ``algorithm`` (string, default 'sage') -- choice of
+        - ``algorithm`` (string, default ``'sage'``) -- choice of
           implementation (for real embeddings only) between 'sage'
```
Comment (GiacomoPope) [inline src/sage/schemes/elliptic_curves/period_lattice.py]:
```suggestion
          implementation (for real embeddings only) between ``'sage'``
```

File: src/sage/schemes/elliptic_curves/period_lattice.py (hunk 2 above)
Comment (GiacomoPope) [inline src/sage/schemes/elliptic_curves/period_lattice.py]:
```suggestion
        - ``prec`` (integer or ``None`` (default)) -- floating point precision (in bits); if ``None``,
```

File: src/sage/schemes/elliptic_curves/period_lattice.py [hunk 10]
```diff
@@ -713,7 +713,7 @@ def _compute_periods_complex(self, prec=None, normalise=True):
         - `prec` (int or ``None`` (default)) -- floating point precision (in bits); if None,
           use the default precision.
 
-        - `normalise` (bool, default True) -- whether to normalise the
+        - `normalise` (bool, default ``True``) -- whether to normalise the
```
Comment (GiacomoPope) [inline src/sage/schemes/elliptic_curves/period_lattice.py]:
```suggestion
        - ``normalise`` (bool, default ``True``) -- whether to normalise the
```

File: src/sage/schemes/elliptic_curves/period_lattice.py [hunk 11]
```diff
@@ -857,9 +857,9 @@ def real_period(self, prec=None, algorithm='sage'):
         - ``prec`` (int or ``None`` (default)) -- real precision in
```
Comment (GiacomoPope) [inline src/sage/schemes/elliptic_curves/period_lattice.py]:
```suggestion
        - ``prec`` (integer or ``None`` (default)) -- real precision in
```

Comment (GiacomoPope) [reply src/sage/schemes/elliptic_curves/period_lattice.py]: hmm I see that there's many `int` and many `integer` -- now im less confident in whcih we should use here.

File: src/sage/schemes/elliptic_curves/period_lattice.py [hunk 12]
```diff
@@ -977,16 +977,16 @@ def basis_matrix(self, prec=None, normalised=False):
...
         INPUT:
 
-        - ``prec`` (int or ``None``(default)) -- real precision in
+        - ``prec`` (int or ``None`` (default)) -- real precision in
           bits (default real precision if ``None``).
 
-        - ``normalised`` (bool, default False) -- if True and the
+        - ``normalised`` (bool, default ``False``) -- if ``True`` and the
           embedding is real, use the normalised basis (see
           ``normalised_basis()``) instead of the default.
```
Comment (GiacomoPope) [inline src/sage/schemes/elliptic_curves/period_lattice.py]: should we do sphinx syntax here for `normalised_basis()`?

File: src/sage/schemes/elliptic_curves/period_lattice.py [hunk 13]
```diff
@@ -1862,7 +1864,7 @@ def elliptic_exponential(self, z, to_curve=True):
             (0.000000000000000 : 1.00000000000000 : 0.000000000000000)
 
         The elliptic exponential of `z` is returned as (0 : 1 : 0) if
```
Comment (GiacomoPope) [inline src/sage/schemes/elliptic_curves/period_lattice.py]:
```suggestion
        The elliptic exponential of `z` is returned as `(0 : 1 : 0)` if
```

File: src/sage/schemes/hyperelliptic_curves/mestre.py [hunk 14]
```diff
@@ -229,7 +229,7 @@ def Mestre_conic(i, xyz=False, names='u,v,w'):
 
     - ``i`` - list or tuple of length 4 containing the four Igusa-Clebsch
       invariants: I2, I4, I6, I10
-    - ``xyz`` - Boolean (default: False) if True, the algorithm also
+    - ``xyz`` - Boolean (default: ``False``) if ``True``, the algorithm also
       returns three invariants x,y,z used in Mestre's algorithm
     - ``names`` (default: 'u,v,w') - the variable names for the conic
```
Comment (GiacomoPope) [inline src/sage/schemes/hyperelliptic_curves/mestre.py]:
```suggestion
     returns three invariants `x`,`y`,`z` used in Mestre's algorithm
    - ``names`` (default: ``'u,v,w'``) - the variable names for the conic
```

**Review by GiacomoPope — COMMENTED** [review]
I left a bunch of suggestions and small comments thoughout. Essentially everything you've done improves the docstrings but I noticed some missing changes. I probably missed even more changes as I only reviewed the diff.

File: src/sage/schemes/elliptic_curves/kraus.py (hunk 5 above)
Comment (mkoeppe) [reply src/sage/schemes/elliptic_curves/kraus.py]: Probably formatting as LaTeX would be good here, but I'm not going to do it in this PR.
I'll be happy to review a follow-up PR that makes further improvements.

File: src/sage/schemes/elliptic_curves/kraus.py (hunk 8 above)
Comment (mkoeppe) [reply src/sage/schemes/elliptic_curves/kraus.py]: That would have to be either
```
 `c_4,c_6`
```
or
```
 `c_4`, `c_6`
```

Follow-up PR?

File: src/sage/schemes/elliptic_curves/period_lattice.py (hunk 11 above)
Comment (mkoeppe) [reply src/sage/schemes/elliptic_curves/period_lattice.py]: I prefer "integer".

Best to take care of in a follow-up just focused on this change and apply it systematically

File: src/sage/schemes/elliptic_curves/period_lattice.py (hunk 12 above)
Comment (mkoeppe) [reply src/sage/schemes/elliptic_curves/period_lattice.py]: yes

Comment (mkoeppe) [reply src/sage/schemes/elliptic_curves/period_lattice.py]: done in dab424b4a68

File: src/sage/schemes/elliptic_curves/kraus.py (hunk 5 above)
Comment (mkoeppe) [reply src/sage/schemes/elliptic_curves/kraus.py]: Actually I found a moment to take care of some of these markup changes in this file. That's 74c43cd9f82

File: src/sage/schemes/hyperelliptic_curves/mestre.py [hunk 15]
```diff
@@ -229,9 +229,9 @@ def Mestre_conic(i, xyz=False, names='u,v,w'):
 
     - ``i`` - list or tuple of length 4 containing the four Igusa-Clebsch
       invariants: I2, I4, I6, I10
-    - ``xyz`` - Boolean (default: False) if True, the algorithm also
-      returns three invariants x,y,z used in Mestre's algorithm
-    - ``names`` (default: 'u,v,w') - the variable names for the conic
+    - ``xyz`` - Boolean (default: ``False``) if ``True``, the algorithm also
+     returns three invariants `x`,`y`,`z` used in Mestre's algorithm
```
Comment (GiacomoPope) [inline src/sage/schemes/hyperelliptic_curves/mestre.py]:
```suggestion
      returns three invariants `x`,`y`,`z` used in Mestre's algorithm
```

File: src/sage/schemes/elliptic_curves/ell_point.py [hunk 16]
```diff
@@ -3274,14 +3274,14 @@ def non_archimedean_local_height(self, v=None, prec=None,
 
         - ``v`` -- a non-archimedean place of `K`, or None (default).
```
Comment (GiacomoPope) [inline src/sage/schemes/elliptic_curves/ell_point.py]:
```suggestion
        - ``v`` -- a non-archimedean place of `K`, or ``None`` (default).
```

File: src/sage/schemes/elliptic_curves/kraus.py (hunk 5 above)
Comment (GiacomoPope) [inline src/sage/schemes/elliptic_curves/kraus.py]:
```suggestion
    The elliptic curve with a-invariants `[0,0,0,-c_4/48,-c_6/864]`, whose
    c-invariants are the given `c_4`, `c_6`.  If the supplied invariants are
```

**Review by GiacomoPope — COMMENTED** [review]
Three last things I noticed. One should fix the failing linter test.

**Comment by mkoeppe** [comment]
Thanks for the edits! Good to go now?

**Comment by GiacomoPope** [comment]
Yep! LGTM. Thanks so much to taking the time to clean these things up.

**Comment by mkoeppe** [comment]
Thanks!

**Pull request merged**